pom.xml

```diff
@@ -45,7 +45,7 @@
         <ubi.glibc.version>2.28-211.el8</ubi.glibc.version>
         <ubi.curl.version>7.61.1-25.el8_7.1</ubi.curl.version>
         <!-- ZULU OpenJDK Package Version -->
-        <ubi.zulu.openjdk.version>8.0.362-1</ubi.zulu.openjdk.version>
+        <ubi.zulu.openjdk.version>11.0.18-1</ubi.zulu.openjdk.version>
```

Comment:
@gracechensd @vinnybod Is it right to make such a large version bump on this old branch?  Or should this not be cherry-picked from 7.2.x all the way back here?

Reply:
I think 5.x should stay on Java 8

Reply:
Thanks, that's what I figured.   I'll pull that change from this PR.

So I should move it to the oldest -post branch already on Java 11 and propagate downstream from there, I assume?

Reply:
nvm... already bumped in 6.0.0-post, and if not already pint merged downstream, will be after I pint merge this PR :-)

Reply:
You should either add [ci skip]` to this before running pint merge or you should run pint merge with strategy `ours`.

Reply:
I didn't do that, because apply-pint-merge was already green for that changeset as merged by Grace yesterday... I think we really do need JDK 11 at that version for common-docker 6.0.0-post and newer to build correctly. Please LMK if I'm mistaken!